README.md

```diff
@@ -72,10 +72,48 @@ forc-wallet accounts
 
 ### Sign a transaction
 
-To sign a transaction, you must provide the transaction ID. You can generate a transaction and get its ID using `forc-client`. Signing the transaction once you have the ID is simple:
+To sign a transaction, you can do so with its ID. You can generate a transaction and get its ID using `forc-client`. Signing the transaction once you have the ID is simple:
 
 ```sh
-forc-wallet account <account_index> sign tx <transaction_id>
+forc-wallet account <account_index> sign tx-id <transaction_id>
+```
+
+### Sign arbitrary data
+
+You may sign a string directly:
+
+```sh
+forc-wallet account <account_index> sign string "Blah blah blah"
+```
+
+Or the contents of a file:
+
+```sh
+forc-wallet account <account_index> sign file <path>
+```
+
+You may also sign a hex-encoded byte string:
+
+```sh
+forc-wallet account <account_index> sign hex 0x0123456789ABCDEF
+```
+
+The hex prefix at the beginning of the string is optional, e.g. the following is the same as above:
+
+```sh
+forc-wallet account <account_index> sign hex 0123456789ABCDEF
+```
+
+You can also use the `sign` subcommand directly, e.g. the following is the same:
+
+```sh
+forc-wallet sign --account <account_index> hex 0123456789ABCDEF
```

Comment:
- One question here @mitchmindtree 
- What's the difference between signing with an account (and stepping through the password prompts successfully)  vs signing directly with a private key?
  - I'm guessing the private key is more direct because you have the key on hand? While the account signing is longer, but doesn't require you to remember/expose your private key?

Reply:
There's no difference in the end! When you use the `account` option, we use the password to derive the private key associated with the account, and we end up signing using that private key.

The private key option is simply for signing directly with a private key. I'm not sure of the use-case exactly :sweat_smile: but added the option upon request from a downstream user. Yes my guess is possibly to simplify testing, where the user had exported an account's private key and preferred to use it directly?

Reply:
About signing with private keys directly:

I think the main reason people kept asking for this feature we have multiple ways of wallet creation (through sdks, the wallet extension etc) so they wanted to sign with the private key resulted from some other method without initializing a new wallet from forc-wallet or importing it, for example the browser wallet extension seems to be outputs seed phrase in 12 words but cli wallet generates 24 words and such differences may tunnel them to use private key especially for using more than one method of wallet management.